session/meson.build

```diff
@@ -52,11 +52,12 @@ install_data(
     install_dir: schemadir
 )
 
+vendor_prefix = 'pantheon'
```

Comment:
We need to use a different prefix here or this package will conflict with our other session package